witness/src/Dscp/Snowdrop/Storage/Avlp.hs

```diff
@@ -0,0 +1,252 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE NamedFieldPuns      #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Dscp.Snowdrop.Storage.Avlp
+    (
+      AVLChgAccum
+    , accumFromAVLMap
+    , mkRocksDbAvlActions
+    , AVLRead
+    , AVLWrite
+    , initAVLStorage
+    ) where
+
+import Codec.Serialise
+import qualified Control.Concurrent.STM as STM
+import qualified Data.ByteString as BS
+import qualified Data.Map.Strict as Map
+import qualified Data.Set as Set
+import Data.Typeable (typeOf)
+
+import qualified Data.Tree.AVL as AVL
+
+import Loot.Base.HasLens
+
+import Snowdrop.Core (CSMappendException (..), ChangeSet (..), ChgAccumModifier (..),
+                      IdSumPrefixed (..), Prefix, StateP, StateR, Undo (..), ValueOp (..),
+                      changeSetToList, idSumPrefix)
+import Snowdrop.Execution (DbAccessActions (..), DbModifyActions (..))
+
+import qualified Dscp.DB.Rocks as RDB
```

Comment:
Are these imports obfuscated on purpose? :thinking:

Reply:
I'm repeating my question: can you please somehow make the imports readable at least to some extent? Also, there are comments that are supposed to be removed (`pureDBPlugin`) -- if you want to leave it, leave it with `error` or hide it from export list by prepending `_` to the function name. Maintaining comments is not a nice thing.